src/scss/06_components/navigation/toggle/_profile-block.scss

```diff
@@ -0,0 +1,186 @@
+.profile {
+  @include font('special.navigation.icon_label');
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  justify-content: center;
+  width: 100%;
+
+  .hel-icon {
+    --icon-size: 24px;
+    margin-bottom: $spacing-half;
+
+    @include breakpoint($breakpoint-l) {
+      --icon-size: 28px;
+    }
+  }
+}
+
+.profile__login-link {
+  @include font('special.navigation.icon_label');
+  align-items: center;
+  border-left: 1px solid $color-black-20;
+  display: flex;
+  flex-direction: column;
+  grid-area: profile;
+  grid-column-end: -1;
```

Comment:
As this was an unintuitive trick to get the profile where we needed it to go, maybe we should add a SCSS comment on this row to explain why it's here.

Reply:
Added comments there that explain this :)